erpnext/erpnext_integrations/custom/contact.json

```diff
@@ -0,0 +1,60 @@
+{
+ "custom_fields": [
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "allow_on_submit": 0,
+   "bold": 0,
+   "collapsible": 0,
+   "collapsible_depends_on": null,
+   "columns": 0,
+   "creation": "2019-12-02 11:00:03.432994",
+   "default": null,
+   "depends_on": null,
+   "description": null,
+   "docstatus": 0,
+   "dt": "Contact",
+   "fetch_from": null,
+   "fetch_if_empty": 0,
+   "fieldname": "is_billing_contact",
```

Comment:
@ronelvcabrera why are you adding `is_billing_contact` as a custom field and exporting as fixtures and why not as a standard field in contact doctype

Reply:
@deepeshgarg007 As advised on this PR comment https://github.com/frappe/frappe/pull/8840#issuecomment-558469686